### PR TITLE
Intro training, prediction and runtime scripts for Raspberry Pi

### PR DESCRIPTION
**Pull request by ernuth**
See `README.md` for installation instructions. The `train.py` and `predict.py` scripts should run on any host after installing the required packages and setting up images for training in `assets/train_cropped`. The script `rpi.py` requires a more specific setup on a Raspberry Pi with a camera.

- The `Catomation/train.py` script uses images in subfolders of `assets/train_cropped` to train a classification model. The script outputs the resulting model at `assets/models/latest.h5`. The training is interrupted early if the validation loss stops improving. A performance graph is generated and displayed in a window when possible, otherwise it is saved locally and the web server folder. If the generated model is satisfactory, it should be moved to `assets/models/cropped.h5` for use by other tools.

- The `Catomation/predict.py` library and script provides the function `predict(image) -> String` that accepts an image of an object and returns the predicted class. It uses the model at `assets/models/cropped.h5` and hard-coded category names.
  When invoked directly as a script it predicts the class of the images located in `assets/manual_validation`.

- The `Catomation/rpi.py` script is for cat detection on a Raspberry Pi host with a camera. This program detects movement (using differential images and OpenCV) and predicts what is the object in the movement area (using a custom TensorFlow model). If the detected object is a cat, the full camera image is saved to a web server. In all cases, the cropped image is set aside for future training.

### CODE REVIEW

**Comment by SaraDaneshvar** [comment]
Great job, thank you!
